### PR TITLE
erlagn :)

### PR DESCRIPTION
**Pull request by monarrk**
why do I keep making these

### CODE REVIEW

File: erlang/question.erl [hunk 1]
```diff
@@ -1,6 +1,10 @@
 -module(question).
--export([main/1, question/2]).
+-export([main/1, question/1, question/2]).
 
+question(Prompt) ->
```
Comment (superwhiskers) [inline erlang/question.erl:4]: there needs to be a comment here stating `implementation of the question function in erlang`

File: erlang/question.erl [hunk 2]
```diff
@@ -12,5 +16,7 @@ question(Prompt, Valid) ->
     end.
 
 main(_) -> 
-    Result = question("Foo", ["Bar", "Baz"]),
-    io:format("~s~n", [Result]).
+    A = question("Foo", ["Bar", "Baz"]),
```
Comment (superwhiskers) [inline erlang/question.erl:19]: for consistency, `Foo`, `Bar`, and `Baz` should be lowercase

File: erlang/question.erl [hunk 3]
```diff
@@ -12,5 +16,7 @@ question(Prompt, Valid) ->
...
+    io:format("~s~n", [A]),
+    B = question("Foo"),
+    io:format("~s~n", [B]).
```
Comment (superwhiskers) [inline erlang/question.erl:22]: don't output the result or perform an "accept all inputs" test for consistency with other implementations

**Review by superwhiskers — CHANGES_REQUESTED** [review]
please make the changes outlined in this review. i will approve and merge once done

**Comment by superwhiskers** [comment]
@Brecert has already implemented one, it seems

**Comment by monarrk** [comment]
yeah

**Pull request closed without merging**